src/widgetastic_patternfly/__init__.py

```diff
@@ -246,10 +246,17 @@ class FlashMessages(Widget):
     def __init__(self, parent, locator, logger=None):
         Widget.__init__(self, parent, logger=logger)
         self.locator = locator
+        self.MSG_XPATH = ('.//div[@id="flash_text_div" or '
+                          'contains(@class, "flash_text_div")]/div[contains(@class, "alert")]')
+        self._flash_messages = []
```

Comment:
Up for implementing this as a property with a setter?

Reply:
Actually, why do we need to store this at all?  I think its going to cause us problems with dismissal and stale references.

Reply:
Fixed.